tox.ini

```diff
@@ -31,6 +31,10 @@ ignore =
     # E203 whitespace before :
     E203,
 
+[pytest]
+markers =
+  non_tox: not run on tox
+
```

Comment:
`pytest.ini` doesn't seem to be used when running tests via Tox. This appears to fix the “non-registered marker” warning.